src/api-node/eth/index.ts

```diff
@@ -0,0 +1,13 @@
+import request from "../../tools/request";
+import {TAssetDetails} from "../assets";
+
+export function fetchEthAssetDetails(base: string, ethAssetId: string | string[]): Promise<TAssetDetails | Array<TAssetDetails>>  {
+    ethAssetId = Array.isArray(ethAssetId) ? ethAssetId : [ethAssetId]
+
+    const params = ethAssetId.map(assetId => `id=${assetId}`)
+        .join('&');
+
+    const query = ethAssetId.length ? `?${params}` : '';
```

Comment:
We have a "query" method for creating a query string. The method should be changed for your case.